app/views/_masthead.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render '/shared/select_work_type_modal' if create_work_presenter.many? %>
```

Comment:
@jamesvanmil I'm just doing a code review of this PR.  Can you explain why you removed `if create_work_presenter.many?` from this view?  Both our develop branch and the Hyrax gem still have this conditional included.